libs/langchain-community/src/chat_models/zhipuai.ts

```diff
@@ -67,10 +67,14 @@ interface ResponseChoice {
   message: ChoiceMessage;
 }
 
+interface ErrorResponse {
+  error: BaseResponse;
```

Comment:
Could we call `BaseResponse` something else? How about `ZhipuAIError`?

Reply:
Yes, we can rename BaseResponse. ZhipuAIError sounds like a good alternative.